README.md

```diff
@@ -31,8 +31,13 @@ TODO
 All examples assume a client and/or channel has been created as follows:
 
 ```go
-client := ably.NewRestClient(ably.ClientOptions{ApiKey: "xxxx"})
-channel := client.Channel('test')
+options, err := ably.NewClientOptions("xxx:xxx")
```

Comment:
Do you agree for removing KeyName and KeySecret fields from ClientOptions and have getters instead, which read Key in the runtime?

```go
ops := ably.ClientOptions{Key: "xxx:xxx"}
keyName := opts.KeyName()
keySecret := opts.KeySecret()
```